curp/src/server/cmd_worker/mod.rs

```diff
@@ -156,11 +155,26 @@ async fn worker_as<
             curp.enter_shutdown();
             if let Err(e) = ce.set_last_applied(entry.index) {
                 error!("failed to set last_applied, {e}");
+                return false;
             }
             cb.write().notify_shutdown();
             true
         }
-        EntryData::ConfChange(_) => false, // TODO: implement conf change
+        EntryData::ConfChange(ref conf_change) => {
+            let changes = conf_change.changes().to_owned();
+            if let Err(e) = ce.set_last_applied(entry.index) {
+                error!("failed to set last_applied, {e}");
+                return false;
+            }
+            let res = curp.apply_conf_change(changes);
```

Comment:
It seems the server only swith to new conf change in after sync stage, and the follower node does not switch to new conf immediately after it received a conf change log?

Reply:
Yes, this implementation is not exactly the same as the raft paper, we use similar logic as etcd.

Reply:
In this implementation, how does the leader guarantee there's no overlapped configuration change in etcd?

Reply:
The leader will confirm that the follower has been applied conf change via commit index, Therefore, the leader does not resend conf change, resulting in overlap.